### ISSUE TITLE
B2 Review

### ISSUE THREAD

**Issue by jshreyans551** [jshreyans551/frontend-101#9]
Completed B2(Clock) with JS `setInterval()` function and `Date()` object

**Comment by LakshSingla** [comment]
- [hourHand.style.transform = "rotateZ(" + hours + "deg)";](https://github.com/jshreyans551/frontend-101/blob/e6f0eec17ee42eadb723ad96f54dafb1a59ee3f2/B/2.html#L76) Although template strings are a part of ES6 syntax and we do not encourage using it during the fest sites unless it is transpiled (due to compatibility issues), but you must know about it and how much cleaner looking code it produces. 

- [line 72-74](https://github.com/jshreyans551/frontend-101/blob/master/B/2.html#L72-L74) Although the code performs well, but it is not readable enough. It may cause trouble when working in a team and other person would wanna make edits to your code. You can make it more readable by adding comments or naming the constants in your code

Apart from these, the logic part of the code is well written :tada: :tada:.

**Comment by jshreyans551** [comment]
Yeah I got what you mean. And I'll add comments for [line 72](https://github.com/jshreyans551/frontend-101/blob/2e18b8bc4c374e3a098aff6ba1704ece0f31718b/B/2.html#L72-L74). Thanks.

**Comment by kunal-mohta** [comment]
@LakshSingla This seams fine now?
